api/notd/model.py

```diff
@@ -66,10 +66,10 @@ class RetrievedTokenTransfer:
     value: int
     gasLimit: int
     gasPrice: int
-    gasUsed: int
+    gasUsed: Optional[int]
     blockNumber: int
-    blockHash: str
-    blockDate: datetime.datetime
+    blockHash: Optional[str]
+    blockDate: Optional[datetime.datetime]
```

Comment:
doesnt this break some apis? have you tested everything locally?

Reply:
ah it would break if something was actually filled as null - can you test that case please.